server/models/requests.coffee

```diff
@@ -10,7 +10,7 @@ module.exports =
 
     contact:
         all: cozydb.defaultRequests.all
-        byName: (doc) ->
+        mailByName: (doc) ->
```

Comment:
DS should handles conflict resolution. Not sure if we should also prefix in each app for additional safety in dev mode. @poupotte @frankrousseau , any opinion ?